dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -204,16 +208,12 @@ public String rssFeed() {
             RSSEntry entry = new RSSEntry();
             if (dbEntry instanceof Workflow) {
                 Workflow workflow = (Workflow)dbEntry;
-                Optional<WorkflowVersion> max = workflow.getWorkflowVersions().stream().filter(v -> v.getDbUpdateDate() != null)
```

Comment:
Does this have the potential to make a big improvement in performance, or are versions fetched anyway?

Reply:
No idea, the there was just a warning that `max` wasn't being used for anything and didn't seem to have any side-effects